doc/classes/Node3D.xml

```diff
@@ -18,97 +18,102 @@
 			<return type="void" />
 			<param index="0" name="gizmo" type="Node3DGizmo" />
 			<description>
-				Attach an editor gizmo to this [Node3D].
+				Attaches the given [param gizmo] to this node. Only works in the editor.
 				[b]Note:[/b] The gizmo object would typically be an instance of [EditorNode3DGizmo], but the argument type is kept generic to avoid creating a dependency on editor classes in [Node3D].
 			</description>
 		</method>
 		<method name="clear_gizmos">
 			<return type="void" />
 			<description>
-				Clear all gizmos attached to this [Node3D].
+				Clears all [EditorNode3DGizmo] objects attached to this node. Only works in the editor.
 			</description>
 		</method>
 		<method name="clear_subgizmo_selection">
 			<return type="void" />
 			<description>
-				Clears subgizmo selection for this node in the editor. Useful when subgizmo IDs become invalid after a property change.
+				Deselects all subgizmos for this node. Useful to call when the selected subgizmo may no longer exist after a property change. Only works in the editor.
 			</description>
 		</method>
 		<method name="force_update_transform">
 			<return type="void" />
 			<description>
-				Forces the transform to update. Transform changes in physics are not instant for performance reasons. Transforms are accumulated and then set. Use this if you need an up-to-date transform when doing physics operations.
+				Forces the node's [member transform] to update. Fails if the node is not inside the tree.
+				[b]Note:[/b] For performance reasons, transform changes are usually accumulated and applied [i]once[/i] at the end of the frame. The update propagates through children 3D nodes, as well. Therefore, use this method only when you need an up-to-date transform (such as during physics operations).
```

Comment:
```suggestion
				[b]Note:[/b] For performance reasons, transform changes are usually accumulated and applied [i]once[/i] at the end of the frame. The update propagates through 3D node children as well. Therefore, only use this method when you need an up-to-date transform (such as during physics operations).
```

Reply:
If it needs to say "3D node children" it may as well say "Node3D children"

Reply:
Can also be "child 3D nodes" but I know you prefer the former :) it just shouldn't be "children ... Nodes"

Reply:
Aaah, right. The PR of last time!